cfme/containers/provider/__init__.py

```diff
@@ -116,30 +110,17 @@ class ContainersProvider(BaseProvider, Pretty):
     page_name = "containers"
     detail_page_suffix = 'provider_detail'
     edit_page_suffix = 'provider_edit_detail'
-    refresh_text = "Refresh items and relationships"
     quad_name = None
     db_types = ["ContainerManager"]
-    _properties_region = prop_region  # This will get resolved in common to a real form
-    add_provider_button = deferred_verpick(
-        {version.LOWEST: form_buttons.FormButton("Add this Containers Provider"),
-         '5.6': form_buttons.add})
-    save_button = deferred_verpick(
-        {version.LOWEST: form_buttons.save,
-         '5.6': form_buttons.angular_save})
+    endpoints_form = ContainersProviderEndpointsForm
 
     def __init__(
             self,
             name=None,
             credentials=None,
             key=None,
             zone=None,
-            hawkular=None,
```

Comment:
The hostname/ipaddress are found in the endpoints so I think it's redundant to add them under the provider class, but, we can add getters only that will take these parameters from the default endpoint. i.e.
`
@property
def hostname(self):
    return self.endpoints.default.hostname
`